src/extensions/ContentDiff/shared/helpers.js

```diff
@@ -0,0 +1,25 @@
+import _ from 'lodash';
+
+export const helpers = () => {};
+
+export const arrayListTestId = "cdd-array-list";
+export const arrayListItemTestId = "cdd-array-list-item";
+
+export const createAssetHtml = (asset) => {
```

Comment:
Separate these functions into 2 separate files under a "helpers" folder.    There should be an index.js in helpers so you can import each one as needed.